src/System.Linq.Expressions/tests/BinaryOperators/Arithmetic/BinaryNullableSubtractTests.cs

```diff
@@ -242,7 +242,7 @@ private static void VerifyNullableShortSubtract(short? a, short? b, bool useInte
                     Enumerable.Empty<ParameterExpression>());
             Func<short?> f = e.Compile(useInterpreter);
 
-            Assert.Equal((short?)(a - b), f());
+            Assert.Equal(unchecked((short?)(a - b)), f());
```

Comment:
Yep. Generally speaking, I'm fixing overflow exceptions. There have been one or two instances total so far where I added an obviously missing unchecked I didn't hit but I'm very conservative with that.